docs/ref/models/database-functions.txt

```diff
@@ -1168,6 +1171,16 @@ It can also be registered as a transform. For example::
     >>> # Get vectors whose round() is less than 20
     >>> vectors = Vector.objects.filter(x__round__lt=20, y__round__lt=20)
 
+.. admonition:: Negative values for ``places`` argument.
+
+    All core database backends, except for SQLite, support passing a negative
+    value for the ``places`` argument. This allows for digits to the left of
+    the decimal point to be rounded to zero.
```

Comment:
Python's `round()` also supports the negative value, so we could fix SQLite by overriding its built-in `ROUND()`. I'm not sure we should, however, so haven't. People can easily override it in their own projects... Not sure if it is worth expanding this to explain how?